README.md

```diff
@@ -290,6 +290,31 @@ labels = PERSON,ORGANISATION,LOCATION
 path = "ner_examples.yml"
 ```
 
+#### spacy.SpanCat.v1
+
+The built-in SpanCat task is a simple adaptation of the NER task to
+support overlapping entities and land its annotations with `doc.spans`.
+
+```ini
+[components.llm.task]
+@llm_tasks = "spacy.SpanCat.v1"
+labels = PERSON,ORGANISATION,LOCATION
+examples = null
+```
+
+| Argument                  | Type                                    | Default      | Description                                                                                                                                  |
+| ------------------------- | --------------------------------------- | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| `labels`                  | `str`                                   |              | Comma-separated list of labels.                                                                                                              |
+| `span_key`                | `str`                                   | `"sc"`       | Key of the `Doc.spans` dict to save the spans under.                                                                                         |
```

Comment:
I'm not sure where the naming needs to modified, but this should be `spans_key` not `span_key`.

Reply:
Ooh nice catch